### DISCUSSION TITLE
李多慧風潮

### DISCUSSION THREAD

**Discussion by poweichen00** [1101-GS4719-C/Python-Community-GS4719-C#1277] [General]
李多慧來臺所帶來的票房是屬於正面效應？
球團為了吸引更多觀眾來看球賽，而風潮過了並沒有對於中職有更好的發展。
但我還是非常支持李多慧，這是她的IG：https://www.instagram.com/le_dahye/

**Comment by harrycoconut** [comment]
李多慧的理想型1.要會顧家2.相處合得來3.能接受他的個性
大家可以審視一下自己有沒有符合喔

**Comment by tower20021202** [comment]
以商業化的角度來看，李多慧身為啦啦隊隊員帶來的票房讓原本中職球隊可說是如虎添翼，但玩影片就已經成為他的粉絲了，好期待下次他來台灣

**Comment by Chiuhsianglin** [comment]
但我覺得李多慧來中職一定有讓更多原本沒有關注棒球的人了解到這個運動，對於台灣的運動也是有正面的影響

**Comment by Chienyuuu6** [comment]
我認為台灣女籃也需要有啦啦隊的機制來吸引觀眾進場看球，自從2017世大運過後的女籃熱潮好像就停滯ㄌ～
不知道大家覺得來個李多慧比較能吸引女籃觀眾還是來個李棟旭🤤🤤

**Comment by PaulSu0905** [comment]
雖然李多慧來台的確能夠給球團帶來可觀的票房收入，但是當大家的目光都專注在啦啦隊而不是場上球員時，是否也會限縮球員的發展空間，導致觀眾完全不關注球賽內容，降低球員競爭的企圖心

**Comment by ziihui** [comment]
多慧的的表演真的超讚！她的舞蹈力度和技巧非常厲害，我覺得她甚至超越了很多女團🥹
看了她的表演後，我也能夠明顯感受到台灣和韓國舞蹈風格上的差異＞＜

我相信多慧的加入會帶給整個啦啦隊更多的學習和進步機會，激發其他成員的學習熱情和競爭意識！
期待她在五月的演出中再次展現她的才華和魅力🥰😍

**Comment by Jack-1D** [comment]
有了李多慧，我們終於不會再有更多林襄路邊抽菸、參加時尚周等等的無聊新聞了。不得不說李多慧是真的有料的，希望她這次來台能激起啦啦隊文化的進步，讓球賽應援變得更具觀賞性。

**Comment by llh26** [comment]
首秀的那陣子，我看了不少李多慧的表演影片，她的舞蹈實力真的高出台灣啦啦隊一大截，表演時總是帶著微笑，還有人拍到他在場下也一直在練習，這種精神真是太讓人佩服了⋯。我認為她的出現對樂天，甚至是中職的啦啦隊都有一定程度的威脅性，期許未來中職的啦啦隊能夠不斷精進自己，讓整體實力提升，吸引更多人願意入場看球賽～（分享一句李多慧影片的留言：「李多慧不像其他人是為了賺錢而跳舞，她是真的熱愛舞蹈！」）

**Comment by Tzuhsuannnnn** [comment]
一開始沒有在關注中職跟啦啦隊，透過最近媒體的大肆宣傳，輾轉看到了多慧的應援影片，看起來超可愛超活潑，也很敬佩他在球場上的那股熱忱。
btw 這是他來台灣的vlog，很可愛，推薦大家去看
:https://www.youtube.com/watch?v=GdZ-kU3_-q0

**Comment by YoYooHuang** [comment]
我覺得發展中華職棒的方式有很多種，像樂天桃猿這種方式也不能說它有錯，畢竟我也看得很開心XD。但如果球迷只是因為啦啦隊才進場看球，那是否失去了欣賞球賽的本質，我覺得也不全然是這樣，球隊也是要賺錢的，這也是他們創造商機的一種方式，若以資方來看這是有正面效益的，不過對於整體棒球風氣有何影響就有待聯盟好好思考了。

**Comment by CPW0919** [comment]
說到李多慧，這個星期六我要去看，好期待喔，我也在這裡推薦他的yt頻道 真的好可愛喔，很適合晚上無聊的時候看心情會變很好!!!
題外話，大家是李多會派還是林襄派，我絕對選李多慧!!!!

**Reply by TW-Maverick** [reply]
我也會選多慧，從她的vlog可以看出她是一個很樂觀的人，笑容非常甜美!
btw 我有去看她的首秀，超讚👍，推薦3公尺長的應援毛巾。

**Comment by Jamesmoneya** [comment]
錯的不是李多惠而是經營棒球隊的球團，很顯然的招募李多惠的球團是樂天，很顯然的他們把很多的重心放在啦啦隊，甚至不惜重金買啦啦隊，也不去改善球場的狀況例如草地排水，球員用品或是訓練用品，這個比啦啦隊重要一百倍，但是樂天還是視而不見，令人很感嘆

**Comment by sjh501** [comment]
不得不說啦啦隊帶來的商機真的不容小覷。雖然說每個人都有選擇要看什麼的自由，職棒也好啦啦隊也好青菜蘿蔔各有所好，但在一些內野的席次出現大隊人拿起手機猛拍是為了啦啦隊而非選手的情形，有時甚至會蓋過比賽本身，還是會令我懷疑這樣的棒球風氣是否值得驕傲和延續。
